smdatatools.py

```diff
@@ -104,24 +111,52 @@
     end_time = time.time()
     print("Elapsed data processing time was %g seconds" % (end_time - start_time))
 
+
```

Comment:
Why add these spaces? If you want to separate sections of code more explicitly, add a comment instead.